### ISSUE TITLE
ProfilesLocation not pulling through from appsettings.json in .net core 2.2

### ISSUE THREAD

**Issue by willroscoe** [aws/aws-logging-dotnet#76]
Using version 1.4.1 of this package.

Configuration.GetAWSLoggingConfigSection(); doesn't pull through 'ProfilesLocation' property from appsettings.json. It is pulling through all the other properties ok. I have tried both forward slashes / and back slashes \ in the path. It looks like in the constructor for  'AWSLoggerConfigSection' in the file: aws-logging-dotnet/src/AWS.Logger.AspNetCore/AWSLoggerConfigSection.cs, 'ProfilesLocation' isn't set.

The appsettings.json I'm using is like this:
{
  "Logging": {
    "LogLevel": {
      "Default": "Trace"
    }
  },
  "AllowedHosts": "*",
  "AWS.Logging": {
    "Profile": "default",
    "ProfilesLocation": "C:\\dev\\aws_credentials.txt",
    "Region": "eu-west-2",
    "LogGroup": "xxxxxxxxxxx,
    "LogLevel": {
      "Default": "Trace",
      "System": "Information",
      "Microsoft": "Information"
    }
  }
}

**Comment by klaytaybai** [comment]
Thanks for reporting this.

**Comment by chazt3n** [comment]
How can I specify the profiles location using Nlog?

**Comment by normj** [comment]
I just added the PR https://github.com/aws/aws-logging-dotnet/pull/90 to add support for ProfileLocation with AWS.Logger.AspNetCore

**Comment by snakefoot** [comment]
@chazt3n You can configure NLog like this:

```xml
<target name="aws" type="AWSTarget" logGroup="NLog.ConfigExample" region="us-east-1" profileLocation="c:\\Users\\sdkuser\\customCredentialsFile.ini" profile="basic_profile" />
```

See also: https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-config-creds.html

**Comment by normj** [comment]
This has been fixed with version 2.0.0 of AWS.Logger.AspNetCore that was released today. https://github.com/aws/aws-logging-dotnet/blob/master/RELEASE.CHANGELOG.md#release-2019-06-27
